nginx.conf

```diff
@@ -0,0 +1,16 @@
+worker_processes auto;
+pid /var/run/nginx/nginx.pid;
+
+events {}
```

Comment:
Is it needed?

Reply:
1. If `worker_processes` is not specified, the default value of worker processes will be set to 1:
![изображение](https://github.com/user-attachments/assets/13b231ae-a402-481a-a8e1-4064a955d4de) 
With parameter auto it will be set to the number of available CPU cores:
![изображение](https://github.com/user-attachments/assets/ff14c2da-7160-4933-bd7e-40fcaeeb3df7)

2. User nginx doesn't have permission to write in the `/var/run/nginx.pid`. In my Dockerfile I set the user `nginx ` as the owner of the `/var/run/nginx` directory, and reassign the `pid `there.

3. Without `events {}` the Nginx webserver won't even start.